language/de/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Erlaubte Herkunft von in Beiträgen verwendeten Bildern',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Alle anderen Bilder werden durch Links ersetzt.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'Die angegebene Bildherkunft ist ungültig.',
+	'EXTIMGASLINK_SECURE_SITES'		=> 'nur Bilder von sicheren Webseiten und dem Server',
+	'EXTIMGASLINK_SERVER_ONLY'		=> 'nur Bilder vom Server',
```

Comment:
Noch ausführlicher als das? Ich würde da wieder an "Die erlaubte Herkunft in Beiträgen verwendeter Bilder" denken, aber das scheint mir nicht wirklich das zu sein, an was du denkst, oder? ^^